serverless.yml

```diff
@@ -48,6 +49,7 @@ provider:
       Resource: "*"
     - Effect: "Allow"
       Action:
+        - "elasticache:Delete*"
```

Comment:
Out of curiosity, is that any reason why we don't use only delete* for all resources we have here ?

Reply:
I know delete is the worst and won't make big difference but just asking out of curiosity :P

Reply:
I'm confused :thinking:  do you mean `elasticache:*` or ower case `elasticache:delete*`?

Reply:
I mean, looks like for all other we use `whatever:*` but for this one we using `elasticache:delete*`. I was just wondering why we don't use `whatever:delete*` as you did for elasticcache. Is there anything that stackjanitor does apart from delete things ? Maybe also scale down in some cases ?

Reply:
Oh yeah - it doesn't need to be this permissive that's true. I'm not too sure why but I'm sure we can eventually lock down the permissions further.
`ec2:*` is definitely concerning haha